setup.py

```diff
@@ -1,36 +1,33 @@
 import os
 from setuptools import setup, find_packages
 
-README = open(os.path.join(os.path.dirname(__file__), 'README.rst')).read()
+README = open(os.path.join(os.path.dirname(__file__), "README.rst")).read()
 
 # allow setup.py to be run from any path
 os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
 
 setup(
-    name='django-model-changes-py3',
-    version='0.16.1',
-    packages=find_packages(exclude=['tests']),
-    license='MIT License',
-    description='django-model-changes allows you to track model instance changes.',
+    name="django-model-changes-py3",
+    version="0.16.2",
+    packages=find_packages(exclude=["tests"]),
+    license="MIT",
```

Comment:
This change, and the one below in `classifiers`, is just in response to some junk output coming through in the Action:
```
/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/setuptools/dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.
!!

        ********************************************************************************
        Please consider removing the following classifiers in favor of a SPDX license expression:

        License :: OSI Approved :: MIT License

        See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
        ********************************************************************************

!!
  self._finalize_license_expression()
```